docs/modules/spark-k8s/partials/nav.adoc

```diff
@@ -9,7 +9,9 @@
 ** xref:spark-k8s:usage-guide/s3.adoc[]
 ** xref:spark-k8s:usage-guide/history-server.adoc[]
 ** xref:spark-k8s:usage-guide/examples.adoc[]
-* xref:spark-k8s:usage-guide/operations/index.adoc[]
-* xref:spark-k8s:usage-guide/operations/pod-placement.adoc[]
-* xref:spark-k8s:usage-guide/operations/pod-disruptions.adoc[]
+** xref:spark-k8s:usage-guide/operations/index.adoc[]
+*** xref:spark-k8s:usage-guide/operations/pod-placement.adoc[]
+*** xref:spark-k8s:usage-guide/operations/pod-disruptions.adoc[]
+*** xref:spark-k8s:usage-guide/operations/pod-disruptions.adoc[]
```

Comment:
There appears to be a duplicate for `pod-disruptions`

Reply:
Good catch, thanks!